WindowsServerDocs/manage/windows-admin-center/azure/azure-extended-network.md

```diff
@@ -0,0 +1,184 @@
+
+## Deploy
+
+Deployment is driven through Windows Admin Center.  
+
+### Install and configure Windows Admin Center
+
+1. [Download and install Windows Admin Center](../understand/windows-admin-center.md) onto any computer capable of running Windows Admin Center other than the two virtual appliances.  
+2. Add Azure Extended Network from the feed.  To do this, select **Settings**:
+![Image](../media/azure-extended-network/settings.png)
+Then select **Extensions**:
+![Image](../media/azure-extended-network/installed-extensions.png)
+In the Available extensions list, select **Azure Extended Network**, then click **Install**.
+After a few seconds you should see a message indicating a successful installation:
+![Image](../media/azure-extended-network/installation-success.png)
+3. Connect your Windows Admin Center deployment to Azure if you haven’t already done so.  If you skip this step now you will be prompted to do so later during the configuration process.
+4. Add a connection to the on-premises virtual appliance.  Return to the connection page and Select **Add** and enter the connection information (machine name and credentials) for the On-premises virtual appliance. Click on the server to connect and you should see Azure Extended Network in the list of tools for this server:
+![Image](../media/azure-extended-network/azure-extended-network.png)
+Click on **Azure Extended Network** to begin.  The first time you will be presented with an overview and a setup button:  
+![Image](../media/azure-extended-network/azure-extended-network-intro.png)
+
+### Deploy Azure Extended network
+
+1. Click **Set up** to begin the configuration.
+2. Click **Next** to proceed past the Overview.
+3. On the Upload Package panel, you will need to download the Azure Extended Network agent package and upload it to the virtual appliance.  Follow the instructions on the panel.  
+
+> [!IMPORTANT] Scroll down if necessary and click **Upload** before you click **Next: Extended-Network Setup**.
+
+4. Select the Subnet CIDR of the On-Premises network that you want to extend.  The list of subnets is read in from the virtual appliance.  If you have not connected the virtual appliance to the correct set of subnets, then you will not see the desired subnet CIDR in this list.
+Click Next after selecting the Subnet CIDR.
+5. Select the Subscription, Resource Group and Virtual Network that you are extending into:
+![Image](../media/azure-extended-network/azure-network.png) 
+The Azure Location and Azure Subnet will be selected automatically.  Click Next: Extended-Network Gateway Setup to proceed.
+6. You will now configure the virtual appliances.  The on-premises gateway should have its information populated automatically:
+![Image](../media/azure-extended-network/azure-network-gateway.png) 
+If it looks correct, you can click Next.
+7. For the Azure virtual appliance you will need to select the Resource Group and VM to use:
```

Comment:
This looks like a sentence expected directly above an image, making the subsequent text lines seem out of order. Is an ending period (dot) intended?
```suggestion
7. For the Azure virtual appliance, you will need to select the Resource Group and VM to use.
```
Also: readability (grammatical) comma.

Reply:
This line was actually missing an image.  I've added that in a separate commit.